### PR TITLE
CSS styling

### PR DESCRIPTION
**Pull request by jmartin777**
# Description

-.homepage-left-container and .homepage-right-container in the .homepage-content-container and applying display: flex; along with flex-direction: row; to it, you'll make both containers appear side by side.

-FieldGuide component, you modified the function signature to accept a prop renderMushroomCards

-updated the CSS in the HomePage component to adjust the layout and styling for the containers.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update
- [ ] Testing
- [x] Refactoring

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/Components/HomePage/HomePage.css [hunk 1]
```diff
@@ -1,9 +1,16 @@
...
+  flex-direction: column;
+  align-items: center;
+}
+
+
+.homepage-flex-container {
+  display: flex;
+  align-items: flex-start;
+}
+
```
Comment (andreasorensen) [inline src/Components/HomePage/HomePage.css:13]: Love love love the styling!

File: src/Components/HomePage/HomePage.css [hunk 2]
```diff
@@ -14,6 +21,7 @@
     box-shadow: 0 14px 19px rgba(0, 0, 0, 0.1);
     color:#624612 ;
```
Comment (Elise-Jones) [inline src/Components/HomePage/HomePage.css:22]: Great styling additions

**Review by andreasorensen — APPROVED** [review]
Excellent job styling!

**Review by Elise-Jones — APPROVED** [review]

File: src/Components/HomePage/HomePage.css (hunk 2 above)
Comment (andreasorensen) [reply src/Components/HomePage/HomePage.css:22]: Agreed!

**Pull request merged**